cell_based/src/writers/population_count_writers/CellProliferativePhasesCountWriter.cpp

```diff
@@ -99,6 +100,12 @@ void CellProliferativePhasesCountWriter<ELEMENT_DIM, SPACE_DIM>::Visit(VertexBas
     VisitAnyPopulation(pCellPopulation);
 }
 
+template<unsigned ELEMENT_DIM, unsigned SPACE_DIM>
```

Comment:
There's something a bit smelly about the amount of copying and pasting of new classes into a big list in these writers. Obviously not something you've come up with or are responsible for @MILeach, but it is making me feel that something weird is going on in the inheritance structure or code organisation here (we are missing the level of abstract class that has this `VisitAnyPopulation` on it, or something?)